lib/containers/common.pm

```diff
@@ -78,7 +79,7 @@ sub install_docker_when_needed {
             }
             else {
                 # We may run openSUSE with DISTRI=sle and openSUSE does not have SUSEConnect
-                if ($host_os =~ 'sles' && script_run("SUSEConnect --status-text | grep Containers") != 0) {
+                if (can_build_sle_base) {
```

Comment:
This might be a problem. Sometimes, the `install_docker_when_needed()` is called from multiple test modules within one test run and you IMO cannot add module which is already added - that why the `SUSEConnect --status-text | grep Containers` command in the condition.

Reply:
At the moment it does not seem to break anything. http://aquarius.suse.cz/tests/4102. But i added it anyway